src/dlgProfilePreferences.cpp

```diff
@@ -258,27 +258,81 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
 
     comboBox_guiLanguage->clear();
     for (auto& code : pMudlet->getAvailableTranslationCodes()) {
-        auto& translatedName = pMudlet->mLanguageCodeMap.value(code).first;
-        int translatedPc = pMudlet->mLanguageCodeMap.value(code).second;
-        comboBox_guiLanguage->addItem(translatedName, code);
-        if (translatedPc >= pMudlet->mTranslationStar) {
-            comboBox_guiLanguage->setItemIcon(comboBox_guiLanguage->count()-1, QIcon(":/icons/rating.png"));
+        auto& translation = pMudlet->mTranslationsMap.value(code);
+        auto& nativeName = translation.getNativeName();
+        if (translation.getIsFromResourceFile()) {
+            auto& translatedPc = translation.getTranslatedPercentage();
+            if (translatedPc >= pMudlet->mTranslationGoldStar) {
+                comboBox_guiLanguage->addItem(QIcon(":/icons/rating.png"),
+                                              nativeName,
+                                              code);
+            } else if (translatedPc >= pMudlet->mTranslationSilverStar) {
+                comboBox_guiLanguage->addItem(QIcon(":/icons/rating_silver.png"),
```

Comment:
Encouragement for those working on translations that are *nearly there* - and *because I could*... :smiling_imp:

Reply:
I don't think it's a good idea. One seems to be quite enough.

Reply:
Just because the Russian translation only gets a Silver-start ATM, perhaps?

Ah well - let me make it look like it was before - I wonder whether the Windows CI build will run the next time around - it was crashing for me on my Win 7 (32bit) PC and I cannot see why....

Reply:
Removed in next commit.